ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -216,6 +217,22 @@ public void send(String message, int id) {
     }
   }
 
+  @ReactMethod
+  public void sendBinary(String message, int id) {
```

Comment:
nit: Can you name this base64String or similar to make it obvious this is Base64 encoded?